book.rb

```diff
@@ -0,0 +1,11 @@
+# Book class
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
```

Comment:
## OPTIONAL 
* I strongly suggest you implement the `a book has many rentals` association by adding A method for adding rentals like you did in the `person class`
```
def add_rental(date, book)
  Rental.new(date, self, person)
end
```